frontend/src/components/activity/content/LAThematicArea.vue

```diff
@@ -1,46 +1,204 @@
-              :uri="contentNode._meta.self"
+            <v-simple-checkbox
+              class="pointer-events-none"
+              :value="parent.hasItem(item)"
+              :color="parent.color"
+              :ripple="false"
             />
           </v-list-item-action>
           <v-list-item-content>
-            <v-list-item-title>{{
-              $tc(`contentNode.laThematicArea.entity.option.${key}.name`)
-            }}</v-list-item-title>
-            <v-list-item-subtitle>{{
-              $tc(`contentNode.laThematicArea.entity.option.${key}.description`)
-            }}</v-list-item-subtitle>
+            <v-list-item-title>{{ item.text }}</v-list-item-title>
+            <v-list-item-subtitle>{{ item.description }}</v-list-item-subtitle>
           </v-list-item-content>
         </v-list-item>
-      </v-list-item-group>
-    </v-list>
+      </template>
+    </e-select>
   </card-content-node>
 </template>
 
 <script>
-import ApiCheckbox from '@/components/form/api/ApiCheckbox.vue'
 import CardContentNode from '@/components/activity/CardContentNode.vue'
 import { contentNodeMixin } from '@/mixins/contentNodeMixin.js'
+import { debounce, isEqual, sortBy } from 'lodash'
+import { serverErrorToString } from '@/helpers/serverError.js'
 
 export default {
   name: 'LAThematicArea',
-  components: { CardContentNode, ApiCheckbox },
+  components: { CardContentNode },
   mixins: [contentNodeMixin],
+  data() {
+    return {
+      localSelection: [],
+      isSaving: false,
+      savingRequestCount: 0,
+      dirty: false,
+      errorMessages: [],
+      debouncedSave: () => null,
+    }
+  },
+  computed: {
+    dataOptions() {
+      return Object.entries(this.contentNode.data.options)
+    },
+    serverSelection() {
+      return this.dataOptions.filter(([_, option]) => option.checked).map(([key]) => key)
+    },
+    selectionCount() {
+      return this.localSelection.length
+    },
+    items() {
+      return this.dataOptions.map(([key, option]) => ({
+        text: this.$tc(`contentNode.laThematicArea.entity.option.${key}.name`),
+        description: this.$tc(
+          `contentNode.laThematicArea.entity.option.${key}.description`
+        ),
+        value: key,
+        checked: option.checked,
+      }))
+    },
+  },
+  watch: {
+    serverSelection: {
+      async handler(newOptions, oldOptions) {
+        if (isEqual(sortBy(newOptions), sortBy(oldOptions))) {
+          return
+        }
+
+        // copy incoming data if not dirty or if incoming data is the same as local data
+        if (!this.dirty || isEqual(sortBy(newOptions), sortBy(this.localSelection))) {
+          this.resetLocalData()
+        }
+      },
+      immediate: true,
+    },
+  },
+  created() {
+    const DEBOUNCE_WAIT = 500
+    this.debounceSave = debounce(this.save, DEBOUNCE_WAIT)
+  },
   methods: {
-    async refreshContent() {
-      await this.api.reload(this.contentNode)
+    onInput() {
+      this.isSaving = true
```

Comment:
do you want to start the saving animation, before the debounceSave kicks in?
In e.g. the ApiTextField, the animation starts as soon as the debounce is over and the saving starts.
If you do that, the isSaving state can be computed from the savingRequestCount (if it is zero).
The you don't have the information if the component is saving duplicated in two properties which you need to synchronize